File: movieApp/app/src/main/java/com/lbc/practice/movieapp/view/second/SecondPresenter.java
package com.lbc.practice.movieapp.view.second;

public class SecondPresenter implements SecondContract.SecondPresent {

    SecondContract.SecondView secondView;

    public SecondPresenter(SecondContract.SecondView secondView) {
        this.secondView = secondView;
    }

    @Override
    public void displayViewPage(int page) {
        secondView.setPage(page);
        secondView.setText(page);
    }
}

File: movieApp/app/src/main/java/com/lbc/practice/movieapp/view/second/SecondPresenter.kt
package com.lbc.practice.movieapp.view.second;

import javax.inject.Inject


class SecondPresenter : SecondContract.SecondPresent {


    @Inject
    constructor()

    var secondView: SecondContract.SecondView?=null

    override fun takeView(secondView: SecondContract.SecondView) {
        this.secondView = secondView
    }
    override fun displayViewPage(page: Int) {
        secondView!!.setPage(page)
        secondView!!.setText(page)
    }
}